contract_compassion/models/contracts.py

```diff
@@ -89,6 +89,9 @@ class RecurringContract(models.Model):
     contract_duration = fields.Integer(compute='_compute_contract_duration',
                                        help='Contract duration in days')
 
+    # Field used to compute days number since last sponsor's letter
+    last_letter = fields.Integer(compute='_compute_last_letter')
```

Comment:
add a `help` parameter describing what the field is instead of using a comment above.